tests/posix/signals/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_POSIX_API=y
+CONFIG_ZTEST=y
+
+CONFIG_POSIX_AEP_CHOICE_BASE=y
```

Comment:
```suggestion
CONFIG_POSIX_AEP_CHOICE_REALTIME_MINIMAL=y
```

Reply:
@cfriedt 
Compliance checks fail with this change but `CONFIG_POSIX_AEP_CHOICE_PSE51=y` passes. 
Can we use that since it selects what is needed? https://docs.zephyrproject.org/latest/kconfig.html#CONFIG_POSIX_AEP_CHOICE_PSE51

Reply:
Yes, sorry, that's right